libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -70,23 +70,116 @@ MRS_API void MRS_CALL mrsCloseEnum(mrsEnumHandle* handleRef) noexcept;
 // Interop
 //
 
+struct mrsAudioTransceiverConfig;
+struct mrsVideoTransceiverConfig;
+struct mrsRemoteAudioTrackConfig;
+struct mrsRemoteVideoTrackConfig;
 struct mrsDataChannelConfig;
 struct mrsDataChannelCallbacks;
 
+// Handles to interop objects (internal implementation)
+
+/// Opaque handle to a native PeerConnection interop object.
+using mrsPeerConnectionHandle = void*;
+
+/// Opaque handle to a native MediaTrack interop object.
+using mrsMediaTrackHandle = void*;
+
+/// Opaque handle to a native AudioTransceiver interop object.
+using mrsAudioTransceiverHandle = void*;
+
+/// Opaque handle to a native VideoTransceiver interop object.
+using mrsVideoTransceiverHandle = void*;
+
+/// Opaque handle to a native LocalAudioTrack interop object.
+using mrsLocalAudioTrackHandle = void*;
+
+/// Opaque handle to a native LocalVideoTrack interop object.
+using mrsLocalVideoTrackHandle = void*;
+
+/// Opaque handle to a native RemoteAudioTrack interop object.
+using mrsRemoteAudioTrackHandle = void*;
+
+/// Opaque handle to a native RemoteVideoTrack interop object.
+using mrsRemoteVideoTrackHandle = void*;
+
+/// Opaque handle to a native DataChannel interop object.
+using mrsDataChannelHandle = void*;
+
+/// Opaque handle to a native ExternalVideoTrackSource interop object.
+using mrsExternalVideoTrackSourceHandle = void*;
+
+// Handles to wrapper objects
+
 /// Opaque handle to the interop wrapper of a peer connection.
 using mrsPeerConnectionInteropHandle = void*;
```

Comment:
nit: I find this thing of two "handles" for each object going through the API really confusing. The fact that the handles for wrappers are called "interop handles" while not being created/managed by the interop library compounds the problem.

IMHO we should either pass just one kind of handles through the interop API, e.g.:
- the interop lib returns a handle that identifies the native object, and the layer above keeps a mapping between this handle and its wrapper
- or, the interop lib takes a wrapper id that identifies the object in the wrapper layer, and keeps a mapping between it and the corresponding native object.

If none of the above is possible, at the very least we should change the naming so that the native "thing" is called differently from the wrapper "thing" and they are not just "handles".

Does not need to be fixed now, happy to move this to an issue to be discussed separately.

Reply:
+1. I expect there's a good reason, but it it pollutes e.g the trackadded signatures.

Reply:
Logged as #207, resolving for now.